tests/testsuite/install.rs

```diff
@@ -2064,7 +2088,22 @@ fn git_install_reads_workspace_manifest() {
 
     cargo_process(&format!("install --git {}", p.url().to_string()))
         .with_status(101)
-        .with_stderr_contains("error: invalid type: integer `3`[..]")
+        .with_stderr_data(str![[r#"
+[UPDATING] git repository `[ROOTURL]/foo`
+[ERROR] invalid type: integer `3`, expected a boolean
+ --> home/.cargo/git/checkouts/foo-[HASH]/[..]/Cargo.toml:6:27
+  |
+6 |             incremental = 3
+  |                           ^
+  |
+[ERROR] invalid type: integer `3`, expected a boolean
+ --> home/.cargo/git/checkouts/foo-[HASH]/[..]/Cargo.toml:6:27
+  |
+6 |             incremental = 3
+  |                           ^
+  |
+
+"#]])
```

Comment:
I'm not sure if the duplicated error message shown here is correct. Therefore, I'm currently avoiding redacting with glob intentionally.

Reply:
CI passes so yeah keep it.